src/scripts/main.js

```diff
@@ -1,3 +1,87 @@
 'use strict';
 
-// write code here
+const head = document.querySelectorAll('thead th');
+const sortElements = Array.from(document.querySelectorAll('tbody tr'));
```

Comment:
The `sortElements` array is created once when the script is loaded. If the table rows change dynamically, this array will not reflect those changes. Consider updating this array inside each sorting function to ensure it always contains the current rows.